smart-contracts/networks.js

```diff
@@ -48,4 +48,9 @@ module.exports = {
     url: 'https://arb1.arbitrum.io/rpc',
     name: 'arbitrum',
   },
+  binance: {
```

Comment:
Maybe eventually we can move all this to the `@unlocl-protocol/networks` package!